packages/arb-token-bridge-ui/src/ui-driver/UiDriverCommon.ts

```diff
@@ -31,3 +35,29 @@ export const stepGeneratorForSmartContractWalletDestinationDialog: UiDriverStepG
       yield* stepGeneratorForDialog('scw_custom_destination_address')
     }
   }
+
+export type UiDriverStepGeneratorForTransaction<
+  TStep extends UiDriverStep = UiDriverStep
+> = (
+  context: UiDriverContext,
+  payload: UiDriverStepPayloadFor<'tx'>
+) => AsyncGenerator<
+  TStep,
+  providers.TransactionReceipt | void,
+  UiDriverStepResultFor<TStep['type']>
+>
+
+export const stepGeneratorForTransaction: UiDriverStepGeneratorForTransaction =
+  async function* (context, payload) {
+    if (context.isSmartContractWallet) {
+      yield* step({ type: 'scw_tooltip' })
+    }
```

Comment:
I've put this into the reusable `stepGeneratorForTransaction` generator function, as we'll need this with every smart contract wallet signing